hack/run-latency-testing.sh

```diff
@@ -1,11 +1,12 @@
 #!/bin/bash
 
-GINKGO_SUITS=${GINKGO_SUITS:-"./test/e2e/performanceprofile/functests/0_config ./test/e2e/performanceprofile/functests/5_latency_testing"}
+GINKGO_SUITS=${GINKGO_SUITS:-"test/e2e/performanceprofile/functests-performance-profile-creator"}
 
 which ginkgo
 if [ $? -ne 0 ]; then
 	echo "Downloading ginkgo tool"
-	go install github.com/onsi/ginkgo/ginkgo
+    go install -mod=mod github.com/onsi/ginkgo/v2/ginkgo@latest
```

Comment:
pin to a specific version please as above (v2.5.1)

Reply:
consider if -mod=mod is needed

Reply:
same